test/server/chat-plugins/help.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Tests for the Help room auto-answerer.
+ * Written by mia-pi.
+ */
+
+'use strict';
+
+const assert = require('assert').strict;
+
+const defaultData = {
+	stats: {},
+	pairs: {},
+	disabled: false,
+	queue: [],
+};
+
+describe('Help', function () {
+	it('should only return true on added regexes', function () {
+		const Answerer = require('../../../.server-dist/chat-plugins/help').HelpAnswerer;
+		const Help = new Answerer(defaultData);
```

Comment:
It's probably better to do `let Help;` at the top and then set this up in a `before()`.

Reply:
Honestly, they all use effectivley the same instance, it could probably just be one constant at the top.

Reply:
Yes, exactly. (I assumed you didn't do that because something needed `Config`, which is what the let; workaround is for -- if it works to just put it in a const, that'd be best.)